### PR TITLE
BZ1586368 Remove references to setting openshift_dns_ip

### PR DESCRIPTION
**Pull request by sdodson**
The node config value dnsIP is no longer configurable in 3.10.

/cc @sjenning

### CODE REVIEW

**Review by kalexand-rh — APPROVED** [review]
LGTM!

**Comment by kalexand-rh** [comment]
@sjenning, will you PTAL?

**Comment by sjenning** [comment]
LGTM thanks

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -299,16 +299,8 @@ the following process for name resolution:
...
-(above the node's nameserver values). That value is defined in the
-*_/etc/origin/node/node-config.yaml_* file by the
-xref:../../install_config/master_node_configuration.adoc#node-configuration-files[`*dnsIP*`]
-parameter, which by default is set to the address of the host node because the host
-is using *dnsmasq*.
-
-. If the `*dnsIP*` parameter is omitted from the *_node-config.yaml_*
-file, then the value defaults to the kubernetes service IP, which is the first
-nameserver in the pod's *_/etc/resolv.conf_* file.
+. {product-title} then sets the pod's first namesever to the IP address of
```
Comment (kalexand-rh) [inline install_config/install/prerequisites.adoc]: s/namesever/nameserver

**Comment by kalexand-rh** [comment]
@jianlinliu, since you approved https://bugzilla.redhat.com/show_bug.cgi?id=1586368, do you agree with this doc change?

@openshift/team-documentation, PTAL

**Comment by jianlinliu** [comment]
LGTM

**Comment by kalexand-rh** [comment]
@sdodson, will you please rebase?

@openshift/team-documentation, PTAL

**Comment by bfallonf** [comment]
LGTM (after typo fix) :bug:

**Pull request closed without merging**